cilib/enums.py

```diff
@@ -86,7 +86,7 @@
     ("1.30", ["1.30/stable", "1.30/candidate", "1.30/beta", "1.30/edge"]),
     ("1.31", ["1.31/stable", "1.31/candidate", "1.31/beta", "1.31/edge"]),
     ("1.32", ["1.32/stable", "1.32/candidate", "1.32/beta", "1.32/edge"]),
-    ("1.33", ["1.33/edge"]),
+    ("1.33", ["1.33/stable", "1.33/candidate", "1.33/beta", "1.33/edge"]),
```

Comment:
This should read like this until `1.33.0` snaps are available.  Otherwise the snap stable channels would have an rc0 in them
```suggestion
    ("1.33", ["1.33/candidate", "1.33/beta", "1.33/edge"]),
```

Reply:
Makes sense. This change was based on this example: https://github.com/charmed-kubernetes/jenkins/pull/1462
I'll update the docs in this PR as well.